mail/common/src/main/java/com/fsck/k9/helper/MailTo.java

```diff
@@ -66,8 +71,23 @@ public static MailTo parse(Uri uri) throws NullPointerException, IllegalArgument
 
         String subject = getFirstParameterValue(params, SUBJECT);
         String body = getFirstParameterValue(params, BODY);
+        String inReplyTo = getFirstParameterValue(params, IN_REPLY_TO);
+
+        String inReplyToMessageId = null;
+        if (inReplyTo != null) {
+            try {
+                List<String> inReplyToMessageIds = MessageIdParser.parseList(inReplyTo);
+                if (inReplyToMessageIds != null && !inReplyToMessageIds.isEmpty()) {
+                    String firstMessageId = inReplyToMessageIds.get(0);
+                    if (firstMessageId != null && !firstMessageId.isEmpty())
+                        inReplyToMessageId = firstMessageId;
+                }
```

Comment:
When `MessageIdParser.parseList()` doesn't throw an exception it always returns a list with at least one valid message identifier. So this code can be simplified to:

```suggestion
                inReplyToMessageId = inReplyToMessageIds.get(0);
```